src/net/java/sip/communicator/impl/protocol/jabber/debugger/SmackPacketDebugger.java

```diff
@@ -131,9 +125,9 @@ public void processStanza(Stanza packet)
 
                     packetLogging.logPacket(
                             PacketLoggingService.ProtocolName.JABBER,
-                            localAddress,
+                            localAddress != null ? localAddress : new byte[4],
```

Comment:
Since you now added the null check inside logPacket, these checks here are unnecessary.

Reply:
Yes, I just force pushed.